test/vfd_swmr_credel.c

```diff
@@ -0,0 +1,483 @@
+/*
```

Comment:
If this is going to be built as a part of the library, it needs to be brought up to THG code standards or it needs to go in the examples.